domains/zeto/build.gradle

```diff
@@ -74,7 +74,7 @@ dependencies {
     coreGo project(path: ":core:go", configuration: "goSource")
 }
 
-task downloadZetoProver {
+task downloadZetoProver(dependsOn: "protoc") {
```

Comment:
Why do these download tasks depend on protoc?

Reply:
must be a result of me experimenting with "outputs" but forgot to remove them after deciding to not use it